lib/top_hangman/io.rb

```diff
@@ -1,55 +1,81 @@
 module TopHangman
   class IO
-    class << self
-      def execute(game, renderer)
-        game.start
+    RUNNING = :running
+    STOPPED = :stopped
 
-        while game.state == Game::RUNNING
-          renderer.show_header(errors_count: game.current_round.errors_count)
+    def initialize
+      @state = RUNNING
+    end
+
+    def execute(game, renderer, file_manager)
+      @file_manager = file_manager
+      game.start
 
-          execute_round_loop(renderer, game) while game.current_round.state == TopHangman::Round::RUNNING
+      while game.state == Game::RUNNING
```

Comment:
Same here, let's create a `Game#running?` method to cover this constant and avoid having to call it everywhere

Reply:
done!